### DISCUSSION TITLE
the check fails regularly because it can't reach packagist

### DISCUSSION THREAD

**Discussion by smknstd** [spatie/security-advisories-health-check#13] [Q&A]
Hello,

This check is great but it keeps crashing when it can't reach packagist:

```
The check named `SecurityAdvisories` did not complete. An exception was thrown with this message: `GuzzleHttp\Exception\RequestException: cURL error 56: OpenSSL SSL_read: error:0A000126:SSL routines::unexpected eof while reading, errno 0 (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://packagist.org/api/security-advisories/`
```

or 
```
The check named `SecurityAdvisories` did not complete. An exception was thrown with this message: `GuzzleHttp\Exception\ServerException: Server error: `POST https://packagist.org/api/security-advisories/` resulted in a `502 Bad Gateway`
```

And it bothers me with false positive alerts:

![Screenshot 2023-07-10 at 14-22-32 Security Advisories - Oh Dear](https://github.com/spatie/security-advisories-health-check/assets/2412608/3fd26dff-7eca-4b51-8025-1c3389cf1b68)

Maybe this package should not crash when there is such an execption. What do you think ?

**Comment by freekmurze** [comment]
Yeah, I'd accept a PR that will mark the test as succeeded if no connection could be made
